bin/start_aiflow.py

```diff
@@ -23,28 +23,7 @@
 import signal
 
 import ai_flow
-
-
-def create_default_sever_config(root_dir_path):
-    """
-    Generate default server config which use Apache Airflow as scheduler.
-    """
-    import ai_flow.config_templates
-    if not os.path.exists(root_dir_path):
-        logging.info("{} does not exist, creating the directory".format(root_dir_path))
-        os.makedirs(root_dir_path, exist_ok=False)
-    aiflow_server_config_path = os.path.join(
-        os.path.dirname(ai_flow.config_templates.__file__), "default_aiflow_server.yaml"
-    )
-    if not os.path.exists(aiflow_server_config_path):
-        raise Exception("default aiflow server config is not found at {}.".format(aiflow_server_config_path))
-
-    aiflow_server_config_target_path = os.path.join(root_dir_path, "aiflow_server.yaml")
-    with open(aiflow_server_config_path, encoding='utf-8') as config_file:
-        default_config = config_file.read().format(**{'AIFLOW_HOME': root_dir_path})
-    with open(aiflow_server_config_target_path, mode='w', encoding='utf-8') as f:
-        f.write(default_config)
-    return aiflow_server_config_target_path
+from ai_flow.util.config_utils import create_default_server_config
```

Comment:
I think we can remove the `start_aiflow.py` and `start_notification.py` completely. And we use the `config init` in the `init-*-env.sh` script.

Reply:
IMO, the `start_aiflow.py` and `start_notification.py` could be removed in another pull request. Removing the `start_aiflow.py` and `start_notification.py` should be based on the `config` and `server` command.

Reply:
We will do that in another pr then. I created issues #202 #203